### ISSUE TITLE
Formatting Problems on the Manage Content Page of Dashboard

### ISSUE THREAD

**Issue by DigitalProf** [BYU-ARCLITE/Ayamel-Examples#641]
The "Name" field needs to be set to a narrower width (and also to wrap) so the "View" buttons are present to enable the viewing of the content.

**Comment by dro248** [comment]
This is the same issue as #637.

**Comment by DigitalProf** [comment]
As an additional comment, it is clear that some of the titles are WAY too long to be meaningful. I corrected a couple, copying some of the title into the comments field, but there several assets (a few chemistry videos for example) for that to be practical. We should restrict the size of the field and wrap the text at that point, or perhaps even limit the length of titles.

Also, watching the page go through its updates reveals a couple of things. I stopped the process mid-stream and got this display, which shows a couple of Resource IDs that are way too long. How can this happen?

![image](https://cloud.githubusercontent.com/assets/7268293/9558845/f7376544-4da7-11e5-8099-47096dcf64f5.png)
